CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
+* `Obj::set_list_values` inappropriately resizes list for LinkList causing LogicError to be thrown. ([#4028](https://github.com/realm/realm-core/issues/4028), since v6.0.0)
```

Comment:
@jedelbo Please write the release notes in wording that's closer to what an end-user would understand and what's usable in SDKs. That will help all SDK's so they don't have all have to reword the same.

Reply:
This isn't something that'll show up in SDK changelogs because it's not something SDKs are currently using.